h3-quinn/Cargo.toml

```diff
@@ -21,6 +21,7 @@ quinn = { version = "0.11", default-features = false, features = [
 tokio-util = { version = "0.7.9" }
 futures = { version = "0.3.28" }
 tokio = { version = "1", features = ["io-util"], default-features = false }
+h3-datagram = { path = "../h3-datagram" }
```

Comment:
Wondering, should it be optional? Is there enough code in here that people would benefit if they didn't need to compile it?

Reply:
> Wondering, should it be optional? Is there enough code in here that people would benefit if they didn't need to compile it?

Changed it.